gremlin/traversal/flows.go

```diff
@@ -886,6 +926,9 @@ func (s *FlowGremlinTraversalStep) Exec(last traversal.GraphTraversalStep) (trav
 		if s.rawpacketsNextStep {
 			return &FlowTraversalStep{GraphTraversal: graphTraversal, Storage: s.Storage, flowSearchQuery: flowSearchQuery}, nil
 		}
+		if s.groupNextStep {
```

Comment:
not sure to understand how it could work as the flowset is filled line 933 ? We skip for metrics and rawpackets because the datastore request will be made there but in your case I don't see where the request will be made ?
I think it doesn't work in history mode. I don't think we can reduce for group or you will have to implement to group thing in ES and OrientDB